scripts/seed.py

```diff
@@ -0,0 +1,210 @@
+
+from datetime import timedelta
+from faker import Factory
+
+import os
+
+from challenges.models import Challenge, ChallengePhase, DatasetSplit, Leaderboard, ChallengePhaseSplit
+from hosts.models import ChallengeHostTeam, ChallengeHost
+from participants.models import Participant, ParticipantTeam
+
+
+def create_admin_user():
+    username = "admin"
+    password = "password"
+    email = "admin@gmail.com"
+    user = User.objects.create_user(
+        email=email,
+        username=username,
+        password=password,
+        is_staff=True,
+        is_superuser=True,
+    )
+    EmailAddress.objects.create(user=user, email=email, verified=True, primary=True)
+    print "Super user created with \n username: %s \n password: %s" % (username, password)
+
+
+def create_user(username=""):
+    Factory.create()
+    password = "password"
+    email = "%s@gmail.com" % (username)
+    user = User.objects.create_user(
+        email=email,
+        username=username,
+        password=password,
+    )
+    EmailAddress.objects.create(user=user, email=email, verified=True, primary=True)
+    print "User created with \n username: %s \n email: %s \n password: %s" % (username, email, password)
+    return user
+
+
+def create_challenge_host_team(user):
+    fake = Factory.create()
+    team_name = "%s Host Team" % (fake.city())
+    team = ChallengeHostTeam.objects.create(
+        team_name=team_name,
+        created_by=user,
+    )
+    print "Challenge Host Team created with \n team_name: %s \n created_by: %s" % (team_name, user.username)
+    ChallengeHost.objects.create(user=user, team_name=team, status="Self", permissions="Admin")
+    print "Challenge Host created with \n user: %s \n team_name: %s" % (user.username, team_name)
+    return team
+
+
+def create_challenges(number_of_challenges=3, host_team=None):
+    fake = Factory.create()
+    for i in xrange(number_of_challenges):
+        if (i % 3 == 0):
+            create_challenge_object("%s Challenge" % (
+                fake.first_name()),
+                timezone.now() - timedelta(days=100),
+                timezone.now() + timedelta(days=500),
+                host_team
+            )
+        elif (i % 3 == 1):
+            create_challenge_object("%s Challenge" % (
+                fake.first_name()),
+                timezone.now() - timedelta(days=500),
+                timezone.now() - timedelta(days=100),
+                host_team
+            )
+        elif (i % 3 == 2):
+            create_challenge_object("%s Challenge" % (
+                fake.first_name()),
+                timezone.now() + timedelta(days=100),
+                timezone.now() + timedelta(days=500),
+                host_team
+            )
+
+
+def create_challenge_object(title, start_date, end_date, host_team):
+    fake = Factory.create()
+    evaluation_script = open(os.path.join(settings.BASE_DIR, 'examples', 'example1', 'string_matching.zip'), 'rb')
```

Comment:
Does this file path really exists?

In case it does, why such a big file path. I may be missing something here so can you please elaborate?